### PR TITLE
PKG: Add NEWS and INSTALL

### PR DESCRIPTION
**Pull request by miked-mellanox**

### CODE REVIEW

**Comment by miked-mellanox** [comment]
@shamisp , @yosefe - please review.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com:8888/jenkins/job/gh-ucx-pr/753/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/190/ for details. (Mellanox internal link)

File: INSTALL [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+
+Developer Builds
+================
+
+shell$ ./autogen.sh
+shell$ ./contrib/configure-devel --prefix=$PWD/install-debug
+
+You will need very recent versions of GNU Autoconf, Automake, and
+Libtool.
```
Comment (shamisp) [inline INSTALL]: Can we be more specific here ?

File: INSTALL [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+This file is a *very* short overview of building and installing UCX
+and building MPI/SHMEM libraries with it.  Much more information is
+available on the UCX web site and github wiki
+
+    http://openucx.org/
+
+    -and-
+
+    https://github.com/openucx/ucx/wiki
```
Comment (shamisp) [inline INSTALL]: I would reference the wiki first and then openucx.org

File: INSTALL [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+serial and does not benefit much from parallelization.
+
+# Build rpm package
+shell$ contrib/buildrpm.sh -s -b
+ 
+# Build deb package
+shell$ dpkg-buildpackage -us -uc
+ 
+# Build Doxygen documentation
+shell$ make docs
```
Comment (shamisp) [inline INSTALL]: I would mention that doxygen has to be installed in order to support "doc" build

File: INSTALL [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+all" as a user with write permissions in the build tree, and a
+separate "make install" as a user with write permissions to the
+install tree.
+
+Compiling support for various networks or other specific hardware may
+require additional command line flags when running configure.
+
+Parallel builds are also supported (although some versions of "make",
+such as GNU make, will only use the first target listed on the command
+line when executable parallel builds).  For example (assume GNU make):
```
Comment (overlookoverthere) [inline INSTALL]: when executing parallel builds).

File: INSTALL [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+shell$ dpkg-buildpackage -us -uc
+ 
+# Build Doxygen documentation
+shell$ make docs
+
+
+Compiling OpenMPI with UCX library
+==================================
+
+Open MPI package includes two public software layers: MPI and OpenSHMEM.
```
Comment (overlookoverthere) [inline INSTALL]: The Open MPI package

File: INSTALL [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+The UCX support in OpenMPI is not upstream yet, it is work in progress.
```
Comment (overlookoverthere) [inline INSTALL]: UCX support in OpenMPI

File: INSTALL [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+Current work on OpenMPI/UCX integration is going for OpenSHMEM API and can be found here:
```
Comment (overlookoverthere) [inline INSTALL]: Current work on OpenMPI/UCX integration is ongoing for the OpenSHMEM API and can be found here:

**Comment by shamisp** [comment]
@overlookoverthere - thanks for comments.

File: INSTALL (hunk 1 above)
Comment (miked-mellanox) [inline INSTALL]: we will put autotools versions check in autogen.sh script instead (as separate PR).

File: INSTALL (hunk 2 above)
Comment (miked-mellanox) [inline INSTALL]: good idea

File: INSTALL (hunk 3 above)
Comment (miked-mellanox) [inline INSTALL]: we will add all pre-requisites tool names and versions in the separate PR when learn all deps.

File: INSTALL (hunk 4 above)
Comment (miked-mellanox) [inline INSTALL]: done

File: INSTALL (hunk 5 above)
Comment (miked-mellanox) [inline INSTALL]: done

File: INSTALL (hunk 6 above)
Comment (miked-mellanox) [inline INSTALL]: done

File: INSTALL (hunk 7 above)
Comment (miked-mellanox) [inline INSTALL]: done

File: INSTALL (hunk 1 above)
Comment (shamisp) [inline INSTALL]: even better

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com:8888/jenkins/job/gh-ucx-pr/754/ for details.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/191/ for details. (Mellanox internal link)

**Comment by miked-mellanox** [comment]
@shamisp approved

**Pull request merged**